src/apps/routes/schemas/tool.clj

```diff
@@ -122,13 +77,6 @@
       (->optional-param :implementation)
       (->optional-param :container)))
 
-(defschema NewTool
-  (assoc Tool
-    :id (describe String "The tool's ID")))
-
-(defschema NewToolListing
```

Comment:
This is used as the response schema for the `GET /apps/{system-id}/{app-id}/tools` endpoint, so it was renamed to `AppToolListing` in cyverse-de/common-swagger-api#15.

`AppToolListing` is now a list of `AppDetailsTool`, which is the same as the deleted `NewTool` above, except `AppDetailsTool` also includes an optional key for `:container` (which is a `ToolListingImage`).

The `GET /apps/{system-id}/{app-id}/tools` endpoint currently doesn't return the `:container` field, but this key is optional, and now the endpoint could return that info in the listing, if required, without having to update the schema further.